module_analysis/models/ir_module_module.py

```diff
@@ -155,8 +153,8 @@ def button_analyse_code(self):
 
             # Update the module with the datas
             values = {}
-            for file_ext, analyses in analysed_datas.items():
-                for k, v in analyses.items():
+            for _, analyses in analysed_datas.items():
```

Comment:
Hi. I know that it is a python convention (to use underscore) however, it is confusing in odoo project where _ is used for translation.

Reply:
You can use as well any variable prefixed with underscore (for example `for _disposable, analyses in ...`). Anyway, in this case I would use instead:
```suggestion
            for analyses in analysed_datas.values():
```

Reply:
Hi, indeed the usage of the variable _ can be avoided.